scripts/util/functions.ts

```diff
@@ -55,3 +55,11 @@ export function* traverseChildren(panel: GenericPanel): Generator<GenericPanel>
 		stack.push(...child.Children());
 	}
 }
+
+export function randomInt(min: number, max: number): number {
+	if (min > max) {
+		[max, min] = [min, max];
+	}
+
+	return Math.floor(Math.random() * (max - min + 1) + min);
+}
```

Comment:
Is this inclusive of min and max? `randomInt(2, 4)`  could be over 4 if `Math.random()` returns 0.9999. Is the `+1` needed?

Reply:
We're flooring so can't be over 4, the possible return values for this are 2, 3 and 4. I think it's more intuitive for a such a function to range over a closed interval than half-open -- `randomInt(2, 4)` returning either 2 or 3 seems odd to me.